packages/smart-wallet/src/walletFactory.js

```diff
@@ -23,6 +23,11 @@ export const privateArgsShape = harden(
   ),
 );
 
+export const customTermsShape = harden({
+  agoricNames: M.not(M.undefined()),
+  board: M.not(M.undefined()),
```

Comment:
Not in this case. The error I'm trying to prevent is that someone neglects to include a term (while developing) or forgets to remove a term (that was once used). I've seen both happen. I've never seen someone pass wrong kind of thing to `agoricNames` or `board`.